scripts/evaluation_script.r

```diff
@@ -0,0 +1,56 @@
+rm(list = ls())
+# read all the R files under R folder #
+R_path <- "R/"
+purrr::map(paste0(R_path, list.files("R/")), source)
```

Comment:
We should never `source()` files in our scripts

The solutions are to have the user `devtools::load()` or to just install the package

Depending on how quickly we are going to trash the R code, I will be more or less picky about this. But in general this is a coding practice that will quickly get you into a lot of trouble, when you start to have conflicting names, etc.